### ISSUE TITLE
Issue with unsupported frame rates

### ISSUE THREAD

**Issue by teekay** [teekay/FLACTools#3]
`FlakeReader` has these bits in the method `decode_frame_header`:

```cs
// custom sample rate
if (sr_code0 < 1 || sr_code0 > 11)
{
   // sr_code0 == 12 -> sr == bitreader.readbits(8) * 1000;
   // sr_code0 == 13 -> sr == bitreader.readbits(16);
   // sr_code0 == 14 -> sr == bitreader.readbits(16) * 10;
   throw new Exception("invalid sample rate mode");
}
```
This means, for example, that files encoded at 11kHz sample rate are not supported.

**Comment by teekay** [comment]
Removing the comments and reading those bits makes the previously unplayable file playable.

**Comment by teekay** [comment]
Fixed in ee2b0455179ade7eb45a80653611d9d28f2a9e4f
